DESCRIPTION

```diff
@@ -62,7 +59,5 @@ Imports:
     gridExtra,
     plotly,
```

Comment:
not yet required?

Reply:
Removed pltly. gridExtra is used in add_annotation